test/correctness/issue_4297.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Halide.h"
```

Comment:
Nit: can we give this test a better filename? Tying it to a GitHub issue number goes against precedent.